internal/service/environment.go

```diff
@@ -23,3 +40,36 @@ func (e *Environment) ListEnvironments(ctx *context.Context, request *environmen
 
 	return response, nil
 }
+
+// CreateEnvironment creates environment
+func (e *Environment) CreateEnvironment(ctx *context.Context, request *environment.CreateEnvironmentRequest) error {
+	conn, requestCtx, err := grpcClient(ctx)
+	if err != nil {
+		return err
+	}
+	client := environment.NewEnvironmentServiceClient(conn)
+	stream, err := client.CreateEnvironment(*requestCtx, request)
+	if err != nil {
+		return err
+	}
+
+	log.Info("Creating environment...")
+	spinner := spinner.New(spinner.CharSets[SpinnerType], SpinnerDelay)
+	spinner.Color(SpinnerColor, SpinnerStyle)
+
+	for {
+		response, err := stream.Recv()
```

Comment:
What happens if `stream.Recv()` does not return anything?

Reply:
According to doc
```
RecvMsg blocks until it receives a message into m or the stream is
done. It returns io.EOF when the stream completes successfully. On
any other error, the stream is aborted and the error contains the RPC
status.
```
Stream will either return a response, error or EOF